en/core-api/cardless-credit.md

```diff
@@ -97,30 +100,89 @@ You will get the API response like the following.
+| payment_type       | The type of payment method used by the customer for the transaction. | String |                                                              |
+| transaction_time   | The date and time at which the transaction occurred.         | String | It is in the format, *YYYY-MM-DD* *HH:MM:SS.*<br>Time zone: Western Indonesian Time (GMT+7) |
+| transaction_status | The status of the transaction.                               | String | For more details, refer to [Transaction Status](/en/after-payment/get-status.md#transaction-status). |
+| fraud_status       | The fraud status of the transaction.                         | String | For more details, refer to [Fraud Status](/en/after-payment/get-status.md#fraud-status). |
+
+</article>
+</details>
+
+?>***Note***: The `redirect_url` attribute for the transaction is received.
+
+#### Status Codes and Errors
 
-?> Read [here to simulate/test success payment](/en/technical-reference/sandbox-test.md#cardless-credit).
+| Code | Description                            | Notes                                                     |
+| ---- | -------------------------------------- | --------------------------------------------------------- |
+| 201  | Successful transaction.                | –                                                         |
+| 400  | The `transaction_details` are missing. | Make sure the `order_id` and `gross_amount` are included. |
+| 413  | Syntax error.                          | Check the syntax.                                         |
+| 500  | Internal system error occurred.        | You can try again later.                                  |
 
-### 3. Create Landing Page After Customer Complete the Payment
-After the customer completes the payment via bank's website, the bank website automatically redirect customer to `Finish Redirect URL` which can be configured on MAP (Merchant Administration Portal). You must [login to MAP](https://account.midtrans.com/login). Go to setting -> configuration, and fill in the `Finish Redirect URL` with your landing page endpoint.
-![Direct Debit Payment Flow](./../../asset/image/coreapi/direct_debit_map.png)
 
-On the Finish Redirect URL script, we need to obtain the response sent to the finish url script. Please make sure the `Finish Redirect URL` endpoint can receive POST. The sample code below are written in native php. Please adjust to your own environment.
+#### 2. Redirecting customer to Cardless Credit Website
+The `redirect_url` retrieved from [Sending transaction data to API](/en/technical-reference/core-api/cardless-credit.md#sending-transaction-data-to-api) is used to redirect the customer to the bank's website.
+The customer is then redirected through server-side redirect, using JavaScript like `window.location=[REDIRECT URL]`, or using HTML link `<a href="[REDIRECT URL]">Pay Here!</a>`.
```

Comment:
> The customer is then redirected ...

That sounds like customer is already auto-redirected, that's not the case. The wording should suggest that **Merchant is the one that should implement the redirection**. Through either one of this methods:

> server-side redirect, using JavaScript like `window.location=[REDIRECT URL]`, or using HTML link `<a href="[REDIRECT URL]">Pay Here!</a>`.